config/environments/production.rb

```diff
@@ -79,8 +79,8 @@
   config.action_mailer.smtp_settings = {
     address: 'smtp.gmail.com',
     port: 587,
-    user_name: Rails.application.credentials.dig(:google_smtp, :email),
-    password: Rails.application.credentials.dig(:google_smtp, :password),
+    user_name: Rails.application.credentials.gmail.address,
+    password: Rails.application.credentials.gmail.password,
```

Comment:
Can we set it up as GMAIL_ADDRESS and GMAIL_PASSWORD

I'm not sure how to do nested env variables on render

Reply:
Please feel free to change anything. Let me know if you can't access the gmail account as well